### PR TITLE
PMM-9632 pg_stat_monitor settings reflect.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by denisok — APPROVED** [review]

File: docs/setting-up/client/postgresql.md [hunk 1]
```diff
@@ -190,6 +190,13 @@ You can now [add the service](#add-service).
 
     You can get a list of other available settings with `SELECT * FROM pg_stat_monitor_settings;`.
 
+    Another useful settings item is
```
Comment (rasika-chivate) [inline docs/setting-up/client/postgresql.md:193]:
```suggestion
    Another useful settings parameter is:
```

File: docs/setting-up/client/postgresql.md [hunk 2]
```diff
@@ -190,6 +190,13 @@ You can now [add the service](#add-service).
...
+    ```ini
+    pg_stat_monitor.pgsm_normalized_query
+    ```
+
+    If value is set to 1 then query acutal values are replaced by placeholders. If value is 0 then you will be able to see examples in QAN.
```
Comment (rasika-chivate) [inline docs/setting-up/client/postgresql.md:198]:
```suggestion
     If the value is set to 1, then query actual values are replaced by placeholders. If the value is 0, you can see the examples in QAN. 
```

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/setting-up/client/postgresql.md [hunk 3]
```diff
@@ -190,6 +190,13 @@ You can now [add the service](#add-service).
 
     You can get a list of other available settings with `SELECT * FROM pg_stat_monitor_settings;`.
 
+    Another useful settings parameter is:
```
Comment (rasika-chivate) [inline docs/setting-up/client/postgresql.md:193]:
```suggestion
    Another important parameter is:
```

File: docs/setting-up/client/postgresql.md [hunk 4]
```diff
@@ -190,6 +190,13 @@ You can now [add the service](#add-service).
...
+    ```ini
+    pg_stat_monitor.pgsm_normalized_query
+    ```
+
+     If the value is set to 1, then query actual values are replaced by placeholders. If the value is 0, you can see the examples in QAN.
```
Comment (rasika-chivate) [inline docs/setting-up/client/postgresql.md:198]:
```suggestion
    If the value is set to 1, the actual query values are replaced by placeholders. If the value is 0, the examples are given in QAN. 
```

Comment (rasika-chivate) [reply docs/setting-up/client/postgresql.md:198]: Can we give a link to QAN where the examples are there?

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/setting-up/client/postgresql.md (hunk 4 above)
Comment (JiriCtvrtka) [reply docs/setting-up/client/postgresql.md:198]: Examples are in detail of specific row, so direct link is always different depends on opened row. If you want we can describe it somehow. Its when you open row details then there is tab called examples.

Comment (JiriCtvrtka) [reply docs/setting-up/client/postgresql.md]: @rasika-chivate Take a look on message above. Is it OK?

Comment (rasika-chivate) [reply docs/setting-up/client/postgresql.md]: I think we should describe it.

Comment (jctvrtka) [reply docs/setting-up/client/postgresql.md]: Examples can be found in QAN details tab example.

Is this OK? Or better ideas how to describe it?

Comment (JiriCtvrtka) [reply docs/setting-up/client/postgresql.md]: Added, resolved.

**Pull request merged**